[cppcms] build failure

**Host Environment**
- OS: linux/debian x64
- Compiler: gcc 8.3

**To Reproduce**
Steps to reproduce the behavior:
`root@unique-gig-2:/home/vcpkg-2020.01# vcpkg install cppcms
-bash: vcpkg: command not found
root@unique-gig-2:/home/vcpkg-2020.01# ./vcpkg install cppcms
The following packages will be built and installed:
    cppcms[core]:x64-linux
Starting package 1/1: cppcms:x64-linux
Building package cppcms[core]:x64-linux...
-- Note: cppcms only supports dynamic library linkage. Building dynamic library.
-- Using cached /home/vcpkg-2020.01/downloads/artyom-beilis-cppcms-b72b19915794d1af63c9a9e9bea58e20a4ad93d4.tar.gz
-- Extracting source /home/vcpkg-2020.01/downloads/artyom-beilis-cppcms-b72b19915794d1af63c9a9e9bea58e20a4ad93d4.tar.gz
-- Using source at /home/vcpkg-2020.01/buildtrees/cppcms/src/20a4ad93d4-26c9d29282
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
-- Building x64-linux-rel
CMake Error at ports/cppcms/portfile.cmake:25 (file):
  file must be called with at least two arguments.
Call Stack (most recent call first):
  scripts/ports.cmake:94 (include)


Error: Building package cppcms:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: cppcms:x64-linux
  Vcpkg version: 2020.01.17-unknownhash
`

**Failure logs** 
-Cut and past the appropriate build messages from the console output.
-Please attach any additional failure logs mentioned in the console output.

**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

--- a/ports/activemq-cpp/CONTROL
+++ b/ports/activemq-cpp/CONTROL
@@ -1,5 +1,5 @@
 Source: activemq-cpp
-Version: 3.9.5-1
+Version: 3.9.5-2
 Build-Depends: apr
 Description: Apache ActiveMQ is the most popular and powerful open source messaging and Integration Patterns server.
-Supports: !uwp
+Supports: !(uwp|linux|osx)

--- a/ports/activemq-cpp/portfile.cmake
+++ b/ports/activemq-cpp/portfile.cmake
@@ -1,8 +1,4 @@
-include(vcpkg_common_functions)
-
-if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
-    message(FATAL_ERROR "${PORT} does not currently support UWP")
-endif()
+vcpkg_fail_port_install( ON_TARGET "uwp" "linux" "osx")
 
 set(VERSION 3.9.5)
 
@@ -42,10 +38,7 @@ vcpkg_build_msbuild(
 
 vcpkg_copy_pdbs()
 
-# Handle copyright
-file(COPY ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/activemq-cpp)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/activemq-cpp/LICENSE.txt ${CURRENT_PACKAGES_DIR}/share/activemq-cpp/copyright)
-
+file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
 
 file(
     COPY

--- a/ports/berkeleydb/CONTROL
+++ b/ports/berkeleydb/CONTROL
@@ -1,5 +1,5 @@
 Source: berkeleydb
-Version: 4.8.30-3
+Version: 4.8.30-4
 Homepage: https://download.oracle.com/
 Description: BDB - A high-performance embedded database for key/value data.
-Supports: !uwp
+Supports: !(uwp|linux|osx)

--- a/ports/berkeleydb/portfile.cmake
+++ b/ports/berkeleydb/portfile.cmake
@@ -1,8 +1,4 @@
-if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
-    message(FATAL_ERROR "${PORT} does not currently support UWP")
-endif()
-
-include(vcpkg_common_functions)
+vcpkg_fail_port_install( ON_TARGET "uwp" "linux" "osx")
 
 vcpkg_download_distfile(ARCHIVE
     URLS "http://download.oracle.com/berkeley-db/db-4.8.30.NC.zip"
@@ -25,5 +21,4 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
-# Handle copyright
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/berkeleydb RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

--- a/ports/cppcms/CONTROL
+++ b/ports/cppcms/CONTROL
@@ -1,5 +1,6 @@
 Source: cppcms
-Version: 1.2.1
+Version: 1.2.1-1
 Homepage: https://github.com/artyom-beilis/cppcms
 Description: CppCMS is a Free High Performance Web Development Framework (not a CMS) aimed at Rapid Web Application Development
 Build-Depends: icu, pcre, openssl, zlib
+Supports: !(linux|osx)

--- a/ports/cppcms/portfile.cmake
+++ b/ports/cppcms/portfile.cmake
@@ -1,4 +1,4 @@
-include(vcpkg_common_functions)
+vcpkg_fail_port_install( ON_TARGET "linux" "osx")
 
 vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY ONLY_DYNAMIC_CRT)
 
@@ -6,7 +6,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO artyom-beilis/cppcms
     REF b72b19915794d1af63c9a9e9bea58e20a4ad93d4
-	SHA512 e99d34d14fbde22be725ac2c0bec069fb584e45c66767af75efaf454ca61a7a5e57434bf86109f910884c72202b8cf98fe16505e7d3d30d9218abd4d8b27d5df
+    SHA512 e99d34d14fbde22be725ac2c0bec069fb584e45c66767af75efaf454ca61a7a5e57434bf86109f910884c72202b8cf98fe16505e7d3d30d9218abd4d8b27d5df
 )
 
 vcpkg_find_acquire_program(PYTHON2)
@@ -15,7 +15,7 @@ get_filename_component(PYTHON2_DIR ${PYTHON2} DIRECTORY)
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
-	OPTIONS -DCMAKE_PROGRAM_PATH=${PYTHON2_DIR} -DUSE_WINDOWS6_API=ON
+    OPTIONS -DCMAKE_PROGRAM_PATH=${PYTHON2_DIR} -DUSE_WINDOWS6_API=ON
 )
 
 vcpkg_install_cmake()
@@ -26,6 +26,4 @@ file(REMOVE ${EXE_DEBUG_FILES})
 file(GLOB EXE_FILES ${CURRENT_PACKAGES_DIR}/bin/*.exe)
 file(REMOVE ${EXE_FILES})
 
-# Handle copyright
-file(COPY ${SOURCE_PATH}/MIT.TXT DESTINATION ${CURRENT_PACKAGES_DIR}/share/cppcms)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/cppcms/MIT.TXT ${CURRENT_PACKAGES_DIR}/share/cppcms/copyright)
+file(INSTALL ${SOURCE_PATH}/MIT.TXT DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

--- a/ports/libusbmuxd/CONTROL
+++ b/ports/libusbmuxd/CONTROL
@@ -1,4 +1,5 @@
 Source: libusbmuxd
-Version: 1.2.185
+Version: 1.2.185-1
 Description: A client library to multiplex connections from and to iOS devices
 Build-Depends: libplist
+Supports: !(uwp|linux|osx)

--- a/ports/libusbmuxd/portfile.cmake
+++ b/ports/libusbmuxd/portfile.cmake
@@ -1,3 +1,5 @@
+vcpkg_fail_port_install( ON_TARGET "uwp" "linux" "osx")
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libimobiledevice-win32/libusbmuxd

